### PR TITLE
Clarifying run config - Remote Server

### PR DESCRIPTION
**Pull request by SyamSundarKirubakaran**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,14 +52,15 @@ While developing the language server and the extension, you don't need to deploy
 - In the debug viewlet, run the launch _Launch Extension - Remote Server_
 - The extension will open a socket on port 3333 and will wait for the JDT language server to connect
 - In Eclipse, run the JDT language server as an Eclipse application. 
-    - Create a debug configuration of type _Eclipse Application_.
-   - in the main tab of the debug configuration set the product to `org.eclipse.jdt.ls.core.product`.
-   - in the Environment tab, define a variable `CLIENT_PORT` with value `3333`.
-   - if your workspace contains 'org.eclipse.jdt.ui', use the Plug-Ins tab in the debug configuration to exclude the plug-in. The presence of 'org.eclipse.jdt.ui' will cause the language server to hang. 
+   - Create a debug configuration of type _Eclipse Application_.
+   - In the main tab of the debug configuration select `Run a Product` instread of `Run an Application`.
```
Comment (fbricon) [inline CONTRIBUTING.md]: instead

**Comment by fbricon** [comment]
Can you please sign-off your commits? See https://github.com/redhat-developer/vscode-java/pull/940/checks?check_run_id=136092626

**Comment by SyamSundarKirubakaran** [comment]
Will do.!

**Comment by fbricon** [comment]
@SyamSundarKirubakaran can you please fix the conflict and sign-off your commit?

**Pull request closed without merging**